sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -487,6 +487,10 @@ object DDLUtils {
     isDatasourceTable(table.properties)
   }
 
+  def isSchemaInferred(table: CatalogTable): Boolean = {
+    table.properties.get(DATASOURCE_SCHEMA_TYPE) == Option(SchemaType.INFERRED.name)
```

Comment:
Thanks! @rxin @jaceklaskowski 

I will not change it because using `contains` will break the Scala 2.10 compiler.